### ISSUE TITLE
Adding multiple accounts.

### ISSUE THREAD

**Issue by teemurutus** [ekutner/home-connect-hass#302]
It would be nice to have a possibility to use multiple accounts with Home Connect Alt.
I got blocked a couple of times so I decided to create separate developer account for each appliance because of the limit.

I already created application credentials for each account (which worked with the not-so-improved version).
The selection of the credentials appear but only one can be in use at a time.

![HCAlt](https://github.com/ekutner/home-connect-hass/assets/118662513/454dd76a-046a-427c-a5aa-b3e2266828d3)

**Comment by ekutner** [comment]
Did you use the "Add entry" option to add another instance of the integration? I never tested that and I have no idea how it would behave but I won't be surprised if it didn't work. There is an implicit assumption in the code that there is only one running instance. I'll take a look when I have some time but this is actually a feature request and a very low priority at that, so can't say when I'll get to it.

**Comment by teemurutus** [comment]
Yes, I used exactly the "Add entry" -button. After selecting second credentials to add, it begins the authorization process, and after successful authorization it gives an empty box (in screenshot). There is no new entry or change to an existing entry.

![image](https://github.com/ekutner/home-connect-hass/assets/118662513/cf9e1998-a559-4045-8419-5b1b9912f2d3)

Thank you for giving this a thought at maybe some point!

**Comment by ekutner** [comment]
>I already created application credentials for each account (which worked with the not-so-improved version).

What did you mean with that sentence? Where and how did it work?

**Comment by teemurutus** [comment]
It is possible to manually add or remove application credentials (OAuth Client ID and secret) for integrations at "<home_assistant_address>/config/application_credentials":

![image](https://github.com/ekutner/home-connect-hass/assets/118662513/4be56b13-58cb-4a96-8548-d3580831a08c)

After manually adding more credentials, you get to choose (first screenshot) which credentials to use when you select "add entry".

With the integration included in Home Assistant, it is possible to use multiple accounts after adding credentials for each account:
![image](https://github.com/ekutner/home-connect-hass/assets/118662513/41e72fa4-9f54-4d4a-9f8d-493ecb3ea517)

**Comment by ekutner** [comment]
Release 1.1.0-b3 adds proper support for multiple accounts by using multiple config entries. The README explains how to use it. Feel free to test it.

**Comment by teemurutus** [comment]
This is very good! Everything works smoothly and I have not faced any trouble with three connections.

**Comment by ekutner** [comment]
Closing as done
